README.md

```diff
@@ -1,47 +1,56 @@
+
 - Public dataset of choice: MNIST
 - Collaborative data science scenarios - Parameters:
-  - Overlap of respective datasets: distinct (by stratifying MNIST figures) vs. overlapping (randomized split)
+  - Overlap of respective datasets: distinct (by stratifying MNIST figures) vs. overlapping (with a randomized split)
   - Size of respective datasets: equivalent vs. different
-  - Number of data partners: 3 databases A, B, C (to be parameterized in future improvements of this experiment)
-- ML algorithm: CNN, not too deep so it can run on CPU
+  - Number of data partners: 3 databases A, B, C is our default scenario, but this is to be parameterized
+- ML algorithm: CNN adapted to MNIST, not too deep so it can run on CPU
 - Distributed learning approach: federated learning (other approaches to be tested in future improvements of this experiment)
 - Contributivity evaluation approach:
   - The approaches we would like to compare (further described in the document linked above):
-    - [done] Shapley value: given the limited number of data partners we consider at that stage it is possible to actually 
-compute it with a reasonable amount of resources
+    - [done] Shapley value: given the limited number of data partners we consider at that stage it is possible to actually compute it with a reasonable amount of resources
     - [done] Performance scores of models trained independently on each node
-    - Truncated Monte Carlo Shapley approximation adapted for datasets instead of individual datum
-    - Federated learning step-by-step
-    - (More approaches to be tested in future improvements of this experiment)
+    - [done] Truncated Monte Carlo Shapley approximation adapted for datasets instead of individual datum (including optimized variants)
+    - [in progress] Information sampling method (including optimized variants: regression importance sampling method, Kriging adaptive importance sampling method)
   - Comparison variables (baseline: Shapley value)
     - Contributivity relative values
     - Computation time
   
 ### Using the code files
-- Define your mock scenario(s) in `simulation_run.py` by changing the values of the suggested parameters of the custom scenario
-- Then execute `simulation_run.py` (starting from the default simulation described, you can adapt here what training, evaluation, contributivity measurement you want to simulate)
-- Consult results in the results files generated
+
+- Define your mock scenario(s) in `config.yml` by changing the values of the suggested parameters of the custom scenario (you can browse more available parameters in `scenario.py`)
+- Clear `results.csv` if you want a clean sheet for starters
+- Then execute `simulation_run.py -f config.yml`
+- Consult results in the `results.csv` generated file or via the `analyse_results.ipynb` notebook
```

Comment:
- A results.csv file will be generated in a new folder called experiments folder.
- you can read this raw `results.csv` file or use  `analyse_results.ipynb` notebook to quickly generate figures.

Reply:
OK, done